pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -191,7 +191,7 @@ func initFSS(ctx context.Context, k8sClient clientset.Interface, controllerClust
 	k8sOrchestratorInstance.informerManager = k8s.NewInformer(k8sClient)
 	// TODO: Restrict the Configmap listener to the CSI namespace.
 	// Refer to https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/464
-	k8sOrchestratorInstance.informerManager.AddConfigMapListener(
+	k8sOrchestratorInstance.informerManager.AddConfigMapListener(ctx, k8sClient, k8sOrchestratorInstance.internalFSS.configMapName, k8sOrchestratorInstance.internalFSS.configMapNamespace,
```

Comment:
Why are we taking name of internal FSS configmap here? We just need namespace right? We need to listen on all configmap changes in that namespace. TKG flavor uses both internal and `csi-feature-states` configmaps.

Reply:
Can we take a common namespace here instead of `internal` FSS configmap namespace? Probably take in CSI namespace

Reply:
> We need to listen on all configmap changes in that namespace. TKG flavor uses both internal and csi-feature-states configmaps.

+1

Reply:
> Can we take a common namespace here instead of internal FSS configmap namespace? Probably take in CSI namespace
We don't a common var which gets stored as CSI namespace. I have introduced a new local var to grab the CSI namespace based on the flavor. See if this is fine(in the latest commit)